client/my-sites/site-settings/related-posts/index.jsx

```diff
@@ -40,8 +40,14 @@ const RelatedPosts = ( {
 						text={ translate(
 							'Automatically displays similar content (related posts) at the end of each post.'
 						) }
-						link="https://jetpack.com/support/related-posts/"
+						link="https://en.support.wordpress.com/related-posts/"
 					/>
+					
+					<p className="related-posts__explanation"> 
```

Comment:
We should be using `FormSettingExplanation` here rather than creating a new class with its own CSS.

E.g.:

https://github.com/Automattic/wp-calypso/blob/859538b6860ab398a7d0851a060637ea46fdc9b5/client/my-sites/site-settings/sitemaps.jsx#L75-L79

Reply:
Do you know if we still plan to move the SDK to the Jetpack repo?
If that's the case, we may want to refrain from importing any calypso components.

Reply:
I think we should be fine to use calypso components, this is the settings section and AFAIK we're not publishing it anywhere else yet. It would be different if that was a Gutenberg block for example 😉

Reply:
thanks for clarifying @tyxla

Reply:
Nice, didn't know that! Switched to that now and updated the PR description. (I'm assuming it should be beneath the actual settings because that's what it seems to do each time it's used so far)